core/translation.cpp

```diff
@@ -32,11 +32,23 @@
 #include "os/os.h"
 
 static const char* locale_list[]={
+"aa", //  Afar
```

Comment:
Note that the language codes without country code were added manually, as they were not in my source. I readded the ones that were already declared + a few ones where it seemed relevant, but this is relatively subjective. We can add more when a need arises.